From: Jesse Barnes <jbarnes@hobbes.virtuousgeek.org>
Date: Wed, 12 Mar 2008 22:44:58 +0000 (-0700)
Subject: Add sysfs backlight support for Fujitsu laptops
X-Git-Url: http://gitweb.freedesktop.org/?p=xorg/driver/xf86-video-intel.git;a=commitdiff;h=e04908853db74dc76718f6bc0dfb23b5ab3c5837

Add sysfs backlight support for Fujitsu laptops

Tested by Dan Williams.
---

--- a/src/i830_lvds.c
+++ b/src/i830_lvds.c
@@ -68,6 +68,7 @@ static char *backlight_interfaces[] = {
     "thinkpad_screen",
     "acpi_video1",
     "acpi_video0",
+    "fujitsu-laptop",
     NULL,
 };
 
